pkg/odo/cli/delete/component.go

```diff
@@ -119,17 +121,33 @@ func (o *ComponentOptions) deleteDevfileComponent() error {
 	appName := "app"
 
 	log.Info("Searching resources to delete, please wait...")
-	isInnerLoopDeployed, resources, err := o.clientset.DeleteClient.ListResourcesToDeleteFromDevfile(devfileObj, appName)
+	isInnerLoopDeployed, devfileResources, err := o.clientset.DeleteClient.ListResourcesToDeleteFromDevfile(devfileObj, appName)
 	if err != nil {
 		return err
 	}
-	if len(resources) == 0 {
+	if len(devfileResources) == 0 {
 		log.Infof("No resource found for component %q in namespace %q\n", componentName, namespace)
 		return nil
 	}
-
+	var remainingResources []unstructured.Unstructured
+	k8sResources, _ := o.clientset.DeleteClient.ListResourcesToDelete(componentName, namespace)
```

Comment:
This is not a part of this PR, but, IMO, this method should have been named `ListKubernetesResourceToDelete` instead. The other method in this interface, `ListResourcesToDeleteFromDevfile`, is clear from its name itself as to which resources it will delete.